### DISCUSSION TITLE
Trying to run Brickstore from QTCreator

### DISCUSSION THREAD

**Discussion by ErfinderDesRades** [rgriebl/brickstore#545] [Q&A]
Hello Everybody,

I'm pretty new in using QTCreator, but have started to learn in order to achive more knowledge about brickstore and ist possibilities. By the way a great software, many thanks to the developers for their efforts and passion.
When I load the brickstore project into QT and try to run it from there (e.g. by clicking the play button) I always get the same message: _"Project ERROR: Please set %INNO_SETUP_PATH% to point to the directory containing the 'iscc.exe' binary."_ 
I checked the whole environment to find out how to solve this issue, but unfortunately without any success.

Could anybody give me some advice how to solve this problem. As I did'nt change anything so far, I'm convinced it's just a simple mistake on my side.

Some hints would be highly appreciated.

Regards from sunny Bremen

**Comment by ErfinderDesRades** [comment]
Really nobody out there who can help?

**Reply by rgriebl** [reply]
Sorry - I thought I had already answered. 
While Inno-Setup isn't really needed for local development, the configure/qmake step still checks its existence. You can either comment out the check in the .pro file or just simply install Inno-Setup to satisfy this check.

**Reply by ErfinderDesRades** [reply]
Thanks a lot for your reply. To be sure not to miss anything I installed the INNO Setup App. After doing so I started the build and got 12 error messages. This prompts me to think that I maybe have downloaded the wrong sourcecode or an incomplete one.
Could you be so kind to tell me where I get the coreect and complete sourcecode?

Thank you so much in davance for your efforts.
